cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/DefaultScheduleBuilder.java

```diff
@@ -182,4 +190,31 @@ public ScheduleBuilder abortIfNotMet() {
     constraints.get(constraints.size() - 1).setWaitUntilMet(false);
     return this;
   }
+
+  /**
+   * Inner class that creates a ScheduleCreationSpec from the deployed properties
+   */
+  public class ScheduleCreationBuilder extends ScheduleCreationSpec {
```

Comment:
Mao and I discussed offline that this can't be done since `DefaultAppConfigurer` in `DefaultScheduleBuilder` are in different packages.